tests/python/frontend/onnx/test_forward.py

```diff
@@ -5638,13 +5638,14 @@ def verify_reverse_sequence(x, sequence_lens, batch_axis, time_axis):
     verify_reverse_sequence(x, sequence_lens, 1, 0)
 
 
+@pytest.mark.parametrize("op_name", ["Gelu", "FastGelu"], scope="session")
 @tvm.testing.parametrize_targets
-def test_gelu(target, dev):
+def test_gelu(target, dev, op_name):
```

Comment:
Let's also extend the tests to check accuracy for `fp16` datatype.

Reply:
done